contrib/platform/src/com/sun/jna/platform/win32/Advapi32Util.java

```diff
@@ -1758,13 +1758,14 @@ public EnumKey(HKEY hKey, int dwIndex) {
 	 * @return A environment block
 	 */
 	public static String getEnvironmentBlock(Map<String, String> environment) {
-		StringBuilder out = new StringBuilder();
+		StringBuilder out = new StringBuilder(environment.size() * 32 /* some guess about average name=value length*/);
```

Comment:
It's not critical and I will take it, but I think you should really consider amending it.